opds_import.py

```diff
@@ -176,17 +180,21 @@ def __repr__(self):
 
 
 class OPDSImporter(object):
+    """ Imports editions and license pools from an OPDS feed.
+    Creates Edition, LicensePool and Work rows in the database, if those 
+    don't already exist.
 
-    """Capable of importing editions from an OPDS feed.
+    Should be used when a circulation server asks for data from 
+    our internal content server, and also when our content server asks for data 
+    from external content servers. 
 
-    This importer should be used when a circulation server
-    communicates with a content server. It ignores author and subject
-    information, under the assumption that it will get better author
-    and subject information from the metadata wrangler.
+    Note: Ignores author and subject information, under the assumption that it will 
+    get better author and subject information from the metadata wrangler.
```

Comment:
I think with the advent of the presentation edition stuff, we can relax this restriction. There's no risk that 'bad' data will overwrite 'good' data.